test/format/lrt.c

```diff
@@ -115,10 +115,12 @@ lrt(void *arg)
 			 * timing out - we know there aren't any resources
 			 * pinned so it should succeed eventually.
 			 */
+			testutil_check(pthread_rwlock_wrlock(&g.txn_lock));
```

Comment:
My understanding of the lock is that it prevents races between timestamps being allocated and the oldest timestamp catching up to current.  Since these calls don't use timestamps, why do they need locking?

Reply:
Agreed, pushed in 043fd79.